scripts/config.ts

```diff
@@ -42,7 +42,7 @@ async function createConfig(pkg: keyof typeof pkgNameMap, format: ModuleFormat)
     bundleName: `${pkgNameMap[pkg]}.${formatExt[format] ?? 'js'}`,
     input: {
       resolve: {
-        tsconfigFilename: normalizePath(path.resolve(__dirname, `../tsconfig.lib.json`)),
+        tsconfigFilename: slashes(path.resolve(__dirname, `../tsconfig.lib.json`)),
```

Comment:
is this a windows thing? if so let's take it out in another PR and I can merge that ASAP.

Reply:
Read your description again, makes sense to have this but let's put it in another PR.

Reply:
I've removed this change from this PR